libzfs.pyx

```diff
@@ -369,7 +369,7 @@ class ZFSException(RuntimeError):
         self.code = code
 
     def __reduce__(self):
-        return (self.__class__, (self.code, self.args))
+        return (self.__class__, (self.code, self.args[0]))
```

Comment:
```suggestion
        return (self.__class__, (self.code, *self.args))
```

Reply:
Either way works. If you feel like updating it though the suggestion is a bit more flexible.